ResearchKit/ActiveTasks/ORKLocationRecorder.m

```diff
@@ -36,16 +36,21 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import "ORKRecorder_Internal.h"
 #import "ORKHelpers_Internal.h"
 
-#import "CLLocation+JSONCodingObject.h"
+#import "CLLocation+ORKJSONDictionary.h"
 
 #import <CoreLocation/CoreLocation.h>
 
 
+NSString * const ORKTotalGPSDistanceKey = @"distance_gps";
+const CLLocationAccuracy ORKLocationRequiredAccuracy = 20;
```

Comment:
It's really only a rough calculation included so that I can show something to the user.